client/components/selectors/selectors.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="selectors.css" />
+  </head>
+
+  <body>
+    <div class="selectors-container">
+      <div class="selector-group">
+        <div class="selector-block">
+          <label for="selector-block" class="selector-label input-label"
+            >Lable</label
+          >
+          <button class="dropdown-button"></button>
```

Comment:
this is usually called `trigger`. So, to keep consistent with your naming we can call this class `select-trigger`.

Reply:
rename class as select-button